dts/bindings/i2c/ti,mspm0-i2c.yaml

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI MSPM0 I2C
+
+compatible: "ti,mspm0-i2c"
+
+include: [i2c-controller.yaml, pinctrl-device.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  pinctrl-0:
+    required: true
+
+  pinctrl-names:
+    required: true
+
+  primary-address:
```

Comment:
I hope this property is added for the I2C target mode address.
If so, NACK! Address for the target is from respective dts.
for example here: https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/refs/heads/main/samples/drivers/i2c/target_eeprom/boards/frdm_ke17z.overlay